src/common/dsp/effect/DualDelayEffect.cpp

```diff
@@ -260,7 +260,7 @@ void DualDelayEffect::init_ctrltypes()
    fxdata->p[0].set_type(ct_envtime);
    fxdata->p[1].set_name("Right");
    fxdata->p[1].set_type(ct_envtime);
-   fxdata->p[2].set_name("Feedback");
```

Comment:
this one seems odd change; it actually is the feedback you are controlling there.

Reply:
Sure, the amount of signal being fed back, no? 

I renamed these while considering eventual implementation of #1559 as well. If automation names are constructed from section names + parameter names, then this works out nicely. Otherwise we'd get Feedback Feedback for automation name if it was kept how it was :)